tutorials/notebooks/example_pytorch_ssdlite_mobilenetv3.ipynb

```diff
@@ -0,0 +1,459 @@
+    "Extract the float model's backcone and head, and construct a torch model the only runs them"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "01e90967-594b-480f-b2e6-45e2c9ce9cee",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "class SDD4Quant(torch.nn.Module):\n",
+    "    def __init__(self, in_sdd, *args, **kwargs):\n",
+    "        super().__init__(*args, **kwargs)\n",
+    "        self.add_module(\"base\", in_sdd)\n",
+    "\n",
+    "    def forward(self, x):\n",
+    "        features = self.base.backbone(x)\n",
+    "\n",
+    "        features = list(features.values())\n",
+    "\n",
+    "        # compute the ssd heads outputs using the features\n",
+    "        head_outputs = self.base.head(features)\n",
+    "        return head_outputs\n",
+    "\n",
+    "\n",
+    "model4quant = SDD4Quant(model)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "08fb59fd-3877-45b4-8529-7f9edb687c69",
+   "metadata": {
+    "tags": []
+   },
+   "source": [
+    "### Extract preproecss and postprocess\n",
+    "\n",
+    "Extract the preprocess and postprocess functions from the float model object, and construct separate preprocess and postprocess functions for the representative dataset and evaluation code\n",
+    "\n",
+    "\n",
+    "Note: the MCT output model flattens the float model output data structure to a list, so the PostProcess manually rebuilds it as the original data structure (a dictionary)."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "ff336c30-56c9-4de8-9c42-6462ddb8d2c0",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "def preprocess(image, targets):\n",
+    "    # need to save the original image sizes before resize for the postprocess part\n",
+    "    targets = {'gt': targets, 'img_size': list(image.size[::-1])}\n",
+    "    image = model.transform([torchvision.transforms.ToTensor()(image)])[0].tensors[0, ...]\n",
+    "    return image, targets\n",
+    "\n",
+    "\n",
+    "class PostProcess:\n",
+    "    def __init__(self):\n",
+    "        self.features = [torch.zeros((1, 1, s, s)) for s in [20, 10, 5, 3, 2, 1]]\n",
+    "\n",
+    "    def __call__(self, head_outputs, image_list, original_image_sizes):\n",
+    "        anchors = [a.to(device) for a in model.anchor_generator(image_list, self.features)]\n",
+    "\n",
+    "        if not isinstance(head_outputs, dict):\n",
+    "            if head_outputs[0].shape[-1] == 4:\n",
+    "                head_outputs = {\"bbox_regression\": head_outputs[0],\n",
+    "                                \"cls_logits\": head_outputs[1]}\n",
+    "            else:\n",
+    "                head_outputs = {\"bbox_regression\": head_outputs[1],\n",
+    "                                \"cls_logits\": head_outputs[0]}\n",
+    "\n",
+    "        detections = model.postprocess_detections(head_outputs, anchors, image_list.image_sizes)\n",
```

Comment:
comment: what is model.postprocess_detections() and model.transform.postprocess() ?